Sources_v3/APIClient/Endpoints/Payloads/ChannelEditDetailPayload_Tests.swift

```diff
@@ -38,4 +38,26 @@ class ChannelEditDetailPayload_Tests: XCTestCase {
         // Assert ChannelEditDetailPayload encoded correctly
         AssertJSONEqual(encodedJSON, expectedJSON)
     }
+    
+    func test_pathParameters() {
+        let payload1: ChannelEditDetailPayload<DefaultExtraData> = .init(
+            type: .messaging,
+            team: nil,
+            members: [.unique],
+            invites: [],
+            extraData: .defaultValue
+        )
+        XCTAssertEqual(payload1.pathParameters, "\(payload1.type)")
+        
+        let cid: ChannelId = .unique
+        let payload2: ChannelEditDetailPayload<DefaultExtraData> = .init(
+            cid: cid,
+            team: nil,
+            members: [],
+            invites: [],
+            extraData: .defaultValue
+        )
+        
+        XCTAssertEqual(payload2.pathParameters, "\(payload2.type)/\(payload2.id!)")
```

Comment:
Please add comments of what you're testing here

Reply:
fixed in 000802cf